### PR TITLE
Code Quality: Fixed issue with clicking search box

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Related #13157

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Open a folder using `Details Layout`
   2. Click on the path bar or searchbox

### CODE REVIEW

**Comment by ferrariofilippo** [comment]
@yaira2 @hishitetsu this should prevent the bug

**Comment by hishitetsu** [comment]
When I single click once on an empty area, it works fine. But when I single click two or more times on an empty area, I can't type in the path bar.

**Comment by ferrariofilippo** [comment]
> When I single click once on an empty area, it works fine. But when I single click two or more times on an empty area, I can't type in the path bar.

I see it, maybe it's sufficient to check if the next item to be focused is the listview, I'm trying this way

**Comment by hishitetsu** [comment]
> (To be sure, if you click twice on the path bar you can use it, right?)

Yes, so this may be a minor issue.

**Comment by ferrariofilippo** [comment]
> maybe it's sufficient to check if the next item to be focused is the listview, I'm trying this way

This works!
Still, when refocusing the listview, it will scroll to the top

File: src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs [hunk 1]
```diff
@@ -427,7 +427,9 @@ private async void FileList_ItemTapped(object sender, TappedRoutedEventArgs e)
 			var shiftPressed = InputKeyboardSource.GetKeyStateForCurrentThread(VirtualKey.Shift).HasFlag(CoreVirtualKeyStates.Down);
 			var item = (e.OriginalSource as FrameworkElement)?.DataContext as ListedItem;
 			if (item is null)
+			{
 				return;
+			}
```
Comment (hishitetsu) [inline src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs:432]: Brackets are no more needed.

**Review by hishitetsu — APPROVED** [review]
LGTM. It certainly doesn't work well with refocusing, but it is still better than as is.

**Comment by ferrariofilippo** [comment]
> It certainly doesn't work well with refocusing, but it is still better than as is.

Why might open an issue to track that

**Review by hishitetsu — APPROVED** [review]
LGTM

**Pull request merged**